staging/src/k8s.io/code-generator/cmd/lister-gen/generators/lister.go

```diff
@@ -213,7 +213,6 @@ func (g *listerGenerator) Namers(c *generator.Context) namer.NameSystems {
 
 func (g *listerGenerator) Imports(c *generator.Context) (imports []string) {
 	imports = append(imports, g.imports.ImportLines()...)
-	imports = append(imports, "k8s.io/apimachinery/pkg/api/errors")
 	imports = append(imports, "k8s.io/apimachinery/pkg/labels")
```

Comment:
Shouldn't these also disappear if we are being explicit everywhere?

Reply:
mea culpa.  I saw these in passing but didn't clean them up. I can do that now.

Reply:
Fixed.